week2/readability/readability.c

```diff
@@ -0,0 +1,84 @@
+#include <cs50.h>
```

Comment:
Your readability solution is good. 👍

You could improve it further by reducing the amount of times you are iterating over the whole text. Right now you're iterating three times over the whole text, once in each function. You could wrap these functions with the for loop, like this:

```
for (int i = 0; i < strlen(s); i++) {
  string text = ...

  count letters(text, i)
  count words(text, i)
  count sentences(text, i)
}
```

You're passing the current index to each function alongside with the text.